dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/tracking/ViewLoadingTimer.kt

```diff
@@ -16,7 +16,12 @@ internal class ViewLoadingTimer {
     }
 
     fun onStartLoading(view: Any) {
-        viewsTimeAndState[view]?.let {
+        val viewLoadingInfo = if (viewsTimeAndState.containsKey(view)) {
```

Comment:
I think I had a reason for not allowing this...I think is related with the `onResume/onPause` . If you do this you will send multiple loadingInfo events for one single View.

Reply:
Well we do send multiple loading because as soon as you hit `onPause`, then the view is stopped. `onResume` will always result into a new View.